<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ART</class>
 <widget class="QMainWindow" name="ART">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>682</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>641</width>
    <height>682</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>641</width>
    <height>682</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iMod Middleware</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>625</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="databaseTab">
     <attribute name="title">
      <string>SQL Database</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Used tabels</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Avaible tabels:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TableDownButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TableUpButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveTableButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddTableButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadTabelsButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>340</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="acceptTableButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetBuffer">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>340</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset buffer</string>
      </property>
     </widget>
     <widget class="QListWidget" name="allTabelListWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="activeListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="kinectTab">
     <attribute name="title">
      <string>Kinect for Windows</string>
     </attribute>
     <widget class="QSlider" name="kinectNearSlider">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>22</width>
        <height>201</height>
       </rect>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>800</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSlider" name="kinectFarSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>22</width>
        <height>201</height>
       </rect>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>800</number>
      </property>
      <property name="value">
       <number>800</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="kinectComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>104</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kinect version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kinect depth range:</string>
      </property>
     </widget>
     <widget class="QLabel" name="kinectNearLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="kinectFarLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="kinect_merge_data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Merge Data</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="oculusTab">
     <attribute name="title">
      <string>Middleware</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Oculus Rift</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>571</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Note: Start the server first to edit these fields</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="filterRiftCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>261</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter ARTData for target wiht ID:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>601</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>By clicking the calibration button the actual view becoms the straight look to the left 
You need a filtertarget to perform a calibration.
The calibration will me made for the first filtertarget.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="oculusResetButton">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>150</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="oculusCalibrationButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>150</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Calibrate Oculus Rift</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="oculus_filter_lineEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>331</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Head-Target ID's, seperated by ';'</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>231</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Send Data to Oculus Rift</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>601</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Middleware Network</string>
      </property>
      <widget class="QPushButton" name="connect_art_button">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>25</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect ARTServer</string>
       </property>
      </widget>
      <widget class="QLabel" name="network_art_state_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ART Server: Not Connected</string>
       </property>
      </widget>
      <widget class="QLabel" name="artfps_label">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>FPS: 0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Oculus Rift Tracking:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="configurationTab">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>621</width>
        <height>361</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>619</width>
         <height>359</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select the data source (Note: &quot;Kinect for Windows&quot; is just avaible in the windows version)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>621</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Server simulator</string>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>45</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start A.R.T Server</string>
     </property>
    </widget>
    <widget class="QLabel" name="frameratelabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Framerate (1-120):</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="debugMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug mode</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="framerateSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connected clients:</string>
     </property>
    </widget>
    <widget class="QLabel" name="connectedClientsLabel">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>601</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>623</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>progressBar</zorder>
   <zorder>tabWidget</zorder>
   <zorder>label</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
